webview/src/plots/components/checkpointPlots/CheckpointPlotsWrapper.tsx

```diff
@@ -42,6 +44,8 @@ export const CheckpointPlotsWrapper: React.FC = () => {
       menu={menu}
       nbItemsPerRow={nbItemsPerRow}
       sectionCollapsed={isCollapsed}
+      changeNbItemsPerRow={changeSize}
```

Comment:
[C] I just realised that updating both Custom and Trend plots is going to collide with #3404

cc @julieg18

Reply:
Yes, there, for sure, will be some collisions. Though I don't think they'll be anything too hard to correct :)